modules/integration_aws-fsx/conf/readme.yaml

```
@@ -0,0 +1,26 @@
+documentations:
+  - name: CloudWatch metrics
+    url: 'https://docs.aws.amazon.com/fsx/latest/WindowsGuide/fsx-windows-metrics.html'
+  - name: CloudWatch guide
+    url: 'https://docs.aws.amazon.com/fr_fr/fsx/latest/WindowsGuide/monitoring-cloudwatch.html'
```

Does this module only apply to FSx for Windows?

If not, maybe also link the other variants:

* https://docs.aws.amazon.com/fsx/latest/LustreGuide/monitoring-cloudwatch.html
* https://docs.aws.amazon.com/fsx/latest/ONTAPGuide/monitoring-cloudwatch.html / https://docs.aws.amazon.com/fsx/latest/ONTAPGuide/file-system-metrics.html / https://docs.aws.amazon.com/fsx/latest/ONTAPGuide/volume-metrics.html
* https://docs.aws.amazon.com/fsx/latest/OpenZFSGuide/monitoring-cloudwatch.html

---

Also, link to original, non translated, version:
```suggestion
  - name: CloudWatch metrics
    url: 'https://docs.aws.amazon.com/fsx/latest/WindowsGuide/fsx-windows-metrics.html'
  - name: CloudWatch guide
    url: 'https://docs.aws.amazon.com/fsx/latest/WindowsGuide/monitoring-cloudwatch.html'
```

---

Unfortunately the FSx variants do not have the same metrics.
May be Il should rename the module's directory to `modules/integration_aws-fsx-windows-file-server/`

---

If so, please rename it indeed!